src/python_testing/TC_MCORE_FS_1_1.py

```diff
@@ -125,7 +125,7 @@ def default_timeout(self) -> int:
     @async_test_body
     async def test_TC_MCORE_FS_1_1(self):
         # TODO this value should either be determined or passed in from command line
-        dut_commissioning_control_endpoint = 0
+        dut_commissioning_control_endpoint = 1
```

Comment:
We will need to get this value some other way we cannot hardcode this. Likely we need this value to come in as a command line argument